testing/microlith-test/integration/helpers.bash

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Let's call this couchbase-helpers.bash to make it obvious when sourcing it.

Reply:
```
❯ docker run --rm -it couchbase/oss-cmos:v1 bash                                                                                                                             
By using this software you accept the licensing agreements that can be found in /licenses/
These can be viewed by running a command like so to provide a custom entrypoint: 'docker run ... cat /licenses/*'
[ENTRYPOINT] Running custom: bash
bash-5.1$ pgrep
BusyBox v1.33.1 () multi-call binary.

Usage: pgrep [-flanovx] [-s SID|-P PPID|PATTERN]
```